client/src/components/SearchBar/index.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
```

Comment:
File name should be SearchBar.js

Reply:
Nah naming it index.js means that you can import it from App.js with 'import SearchBar from './components/SearchBar'. If it was SearchBar.js then you would need './components/SearchBar/SearchBar.js'

Reply:
Not personally a fan of this approach; makes it nearly impossible to search for component files by their name or infer a file's contents from its name but I'll concede it if it's what the team wants

Reply:
Oh okay shoot, I mentioned this file structure a while ago and only got positive reactions so thought it was fine. Feel free to make a refactoring issue. I think for the small size of our repo, perhaps cleanliness of the import statements outweighs searchability? I think for this PR I will sick with the index.js naming for the sake of getting the code in.